demos/python_demos/speech_recognition_demo/README.md

```diff
@@ -0,0 +1,136 @@
+# Speech Recognition Demo
+
+This demo demonstrates Automatic Speech Recognition (ASR) with a pretrained Mozilla\* DeepSpeech 0.6.1 model.
+
+## How It Works
+
+The application accepts
+
+ * Mozilla\* DeepSpeech 0.6.1 neural network in Intermediate Representation (IR) format,
+ * n-gram language model file in kenlm quantized binary format, and
+ * an audio file in PCM WAV 16 kHz mono format.
+
+The app also depends on `ctcdecode_numpy` Python\* module, its installation is described below.
+
+Demo workflow is the following:
+
+ * Initialization: the neural network and language model (LM) are read and initialized.
+An OpenFST trie with allowed word prefixed is build during the initialization of LM scorer.
+ * Audio file is loaded
+ * MFCC audio features are extracted from the audio file
+ * MFCC features are split into chunks, and fed to the neural network. The network outputs per-frame character probabilities.
+ * Per-frame character probabilities and the LM scorer are fed into CTC beam search decoder. It finds the transcription of the audio file.
+
+## Model preparation
```

Comment:
do we really need this section in demo readme if we have model.yml, which do the same steps?

Reply:
Removed / shortened.
Several other (8 out of 24) python demos list processing stages like this, starting with words "The demo workflow is the following".

Reply:
As the rule, demo readme includes MO conversion if it works with model not from OMZ e.g. bert demo: https://github.com/openvinotoolkit/open_model_zoo/tree/master/demos/python_demos/bert_question_answering_demo#supported-models

your model will be added to OMZ with demo, right? Then this section should not contains downloading and conversion. Instead you can provide model.lst where will be specified list of models for downloading by model downloader